### ISSUE TITLE
not_empty rule does not catch empty field

### ISSUE THREAD

**Issue by mkrogemann** [zombor/Validator#5]
When I add the following code to self_contained_validator_spec.rb, it fails. Either a bug or I am misunderstanding the Validator. 

``` ruby
  it 'should return false for empty email' do
    foo = SelfContainedValidator.new(OpenStruct.new(:email => ''))
    foo.valid?.should be_false
  end   
```
